### ISSUE TITLE
stuck on "already logged in" screen after signup

### ISSUE THREAD

**Issue by hrothgar** [isTravis/list-of-links#1]
After creating an account, I tried clicking around to the home page, settings page, etc, but until refreshing the page all I got back was this screen:

![screen shot 2016-10-05 at 9 54 22 am](https://cloud.githubusercontent.com/assets/913747/19116063/31073bbe-8ae2-11e6-9fe6-8f8430b66c17.png)

**Comment by isTravis** [comment]
This will happen if there was a bug on the clientside during the signup process. The client-side routing gets broken if the react event loop gets broken somewhere. If this happens again, could you check the console and see if there are any errors being printed.

**Comment by hrothgar** [comment]
I created my account in exactly the same way I did before, and got the error again. I entered all the signup information and also added a photo, clicked "Save Image", and then the signup button. The first screen I got back was the Already Logged In screen. There were no errors at that point. Then I tried clicking to my profile, and got this error in the console:

    Uncaught TypeError: Cannot read property 'getHostNode' of null
    https://www.listoflinks.co/app.1aacbd3a6c0d14ed306f.js
